ddtrace/appsec/_iast/_taint_tracking/Aspects/Helpers.cpp

```diff
@@ -60,7 +60,8 @@ as_formatted_evidence(const string& text,
 
     for (const auto& taint_range : text_ranges) {
         string content;
-        if (!tag_mapping_mode or tag_mapping_mode.value() == TagMappingMode::Normal) {
+        // tag_mapping_mode.value throws a compile error: 'value' is unavailable: introduced in macOS 10.13
```

Comment:
we can drop these comments if we have 10.13 in the test suite

Reply:
> we can drop these comments if we have 10.13 in the test suite

cibuildwheel compiles for macOS Intel and macOS Apple Silicon. Regarding the macOS version, according to the [cibuildwheel docs](https://github.com/pypa/cibuildwheel?tab=readme-ov-file#what-does-it-do):

> Builds manylinux, musllinux, macOS 10.9+ (10.13+ for Python 3.12+), and Windows wheels for CPython and PyPy.

Given this, we still need to maintain compatibility with 10.13 or lower. I think the comment is valid as a reminder that in the future, we should check cibuildwheel and (maybe) consider rolling back these changes. WDYT?